src/main/java/hudson/plugins/jira/JiraSite.java

```diff
@@ -148,7 +161,7 @@
 
     @DataBoundConstructor
     public JiraSite(URL url, URL alternativeUrl, String userName, String password, boolean supportsWikiStyleComment, boolean recordScmChanges, String userPattern,
-                    boolean updateJiraIssueForAllStatus, String groupVisibility, String roleVisibility, boolean useHTTPAuth) {
+                    boolean updateJiraIssueForAllStatus, String groupVisibility, String roleVisibility, boolean useHTTPAuth, Integer timeout) {
```

Comment:
there is more robust
`@DataBoundSetter`
https://groups.google.com/forum/#!msg/jenkinsci-dev/58-DEvuJZWI/5QrxBZRFJ6IJ
http://stapler.kohsuke.org/apidocs/org/kohsuke/stapler/DataBoundSetter.html